README.md

```diff
@@ -47,9 +47,40 @@ index.createIndex()
 # find k nearest similar images
 # choose policy from 'object' or 'global'. Search results will change accordingly.
 # object: Object level matching. The engine will look for similarity at object level for every object detected in the image.
-# global: Overall similarity using single feature space on the whole image. 
+# global: Overall similarity using single feature space on the whole image.
 similar = index.knnQuery('path/to/query/image', k=10, policy='object')
 ```
+
```

Comment:
Can you move this `Docker` section below last line of example usage?

Reply:
Yeah, I have several updates to the README.md